contracts/rollup/BatchManager.sol

```diff
@@ -34,8 +34,8 @@ contract BatchManager is Parameters {
 
     event NewBatch(address committer, uint256 index, Types.Usage batchType);
     event StakeWithdraw(address committed, uint256 batchID);
-    event BatchRollback(uint256 batchID);
-    event RollbackFinalisation(uint256 totalBatchesSlashed);
+    event RollbackComplete(uint256 startID, uint256 nDeleted);
+    event RollbackPartiallyComplete(uint256 startID, uint256 nDeleted);
```

Comment:
I found that using endID at the second parameter might be confusing.
- If endID represents the ID of the last batch deleted, then it's not possible to represent the case that no batch is deleted.
- It could represent the ID of the highest batch that hasn't been deleted. But a little bit harder to reason than simplly the number of how many batches deleted.